src/connectors/xiami.js

```diff
@@ -8,6 +8,8 @@ Connector.artistSelector = '.music .info .singers';
 
 Connector.trackArtSelector = '.music .cover-link .active img';
 
+Connector.getUniqueID = () => $('.play-bar .content .title').attr('href').split('?')[0].split('/song/')[1];
```

Comment:
@inverse  @naoyeye 

I add codes that @naoyeye 's suggested.
Please check and let know me anything that needs changed :)